tests/omhttp-basic.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# This file is part of the rsyslog project, released under ASL 2.0
+
+#  Starting actual testbench
+. ${srcdir:=.}/diag.sh init
+
+port="$(get_free_port)"
+omhttp_start_server $port
+
+error_file=$(pwd)/$RSYSLOG_DYNNAME.omhttp.error.log
+rm -f $error_file
+
+generate_conf
+add_conf '
+#$DebugLevel 2
+template(name="tpl" type="string"
+	 string="{\"msgnum\":\"%msg:F,58:2%\"}")
+
+module(load="../contrib/omhttp/.libs/omhttp")
+
+if $msg contains "msgnum:" then
+	action(
+		# Payload
+		name="my_http_action"
```

Comment:
just my personal preferrence: I put multiple parameters on one line as it makes it easier to grasp the test in one slurp - but **not** a requirement.